CONTRIBUTING_JS.md

```diff
@@ -406,24 +374,10 @@ NPM uses the `.gitignore` by default, so we have to add a `.npmignore` file to e
 
 ##### Dependency management
 
-We suggest either of these:
-
-- [david-dm](https://david-dm.org/)
-- [greenkeeper](http://greenkeeper.io/) to keep your dependencies up to date.
+We suggest using [david-dm](https://david-dm.org) for tracking dep versions.
 
 Every module below 1.0.0 should use `~` instead of `^`.
 
-##### Pre-Commit
-
-[pre-commit](https://www.npmjs.com/package/pre-commit) helps us check code style run the tests on every commit. In your `package.json`:
-
-```json
-"pre-commit": [
-  "lint",
-  "test"
-]
-```
-
```

Comment:
I think our guideline should be to have a `lint` task run on pre-commit/push. It's a quick thing to run and saves a big round trip waiting for CI to fail. We don't need to mandate the specific module to use.

It appears to be in approval from the team https://github.com/ipfs/js-ipfs/issues/1705